src/test/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataMetricsServiceImplTest.java

```diff
@@ -73,6 +76,14 @@ public void nullMetdataUri() throws Exception {
      */
     @Test
     public void validMetdataUri() throws Exception {
+        
+        Field metricsField = fmMetricsServiceImpl.getClass().getDeclaredField("metadataMetrics");
+        metricsField.setAccessible(true);
```

Comment:
This is a bit wonky, maybe we can discuss (offline or here for provenance) a design change that facilitates testing in a proper way (like introducing an `@Autowired` annotated setter method for this property).